draft-ietf-ppm-dap.md

```diff
@@ -1778,7 +1825,8 @@ Otherwise, the Leader proceeds as follows with each report:
 The Helper begins each step of continuation with a sequence of `state` objects,
 which will be `Continued(prep_state)`, one for each report in the candidate set.
 
-The Helper awaits an HTTP POST request to the aggregation job URI from the
+The Helper awaits an HTTP POST request to
+`{helper}/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}` from the
```

Comment:
Did we decide not to include the step in the path? E.g., `?step=<step>`?) Or did we decide to punt to a future PR?

Reply:
Since the POST body contains the step, a `?step=<step>` parameter is redundant. It's up to us whether we want to keep it in the body or as a query parameter, but I don't think we should do both.

IMO I don't think any particular one is better than the other.